specification/rollup/rollup/RollupRequest.ts

```diff
@@ -31,5 +31,6 @@ export interface Request extends RequestBase {
     index: IndexName
     rollup_index: IndexName
   }
+  /** @identifier rollup */
```

Comment:
Should be named `config`. There's no doc for it (hence the `UserDefinedValue`), but the json spec says that this is "The rollup configuration".